lib/text.rb

```diff
@@ -0,0 +1,14 @@
+class Text
+
+attr_reader :number
+
+  def initialize(number)
+    @number = number
+
+  end
+
+  def send(number)
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `number`. If it's necessary, use `_` or `_number` as an argument name to indicate that it won't be used. You can also write as `send(*)` if you want the method to accept any arguments but don't care about them.